src/dmd/frontend.d

```diff
@@ -14,36 +14,31 @@
  */
 module dmd.frontend;
 
-/**
-Initializes the DMD compiler
+import dmd.dmodule : Module;
+import std.range.primitives : isInputRange, ElementType;
+import std.traits : isNarrowString;
```

Comment:
Thought Phobos is DMD was a no-no ?

Reply:
This file isn't used by dmd at all - only by the _DMD as a library_ project.
Should we rename the file or move it to somwhere else, s.t. this gets clearer?

I don't see any reason not to use Phobos here, it allows to write the high-level wrappers and stops us from re-inventing the wheel and blowing up the code with inferior, probably buggy implementations of Phobos.